plugins/guests/redhat/cap/rhn_register.rb

```diff
@@ -81,7 +81,7 @@ def self.rhn_register_upload_certificate(machine, ui)
             if File.exist?(machine.config.registration.ca_cert)
               # Make sure the provided CA certificate file will be configured
               cert_file_name = File.basename(machine.config.registration.ca_cert)
-              cert_file_content = File.read(machine.config.registration.ca_cert, tmp)
+              cert_file_content = File.read(machine.config.registration.ca_cert)
```

Comment:
@KrE80r Any idea why the `tmp` is in here? I am trying to understand the root cause of the regression. cc @budhrg